### PR TITLE
Download blob as string

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: storage/google_cloud/google_cloud_utils.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from urllib.parse import urlparse
+
+from google.cloud import storage
+
+
+def download_blob_to_string(bucket_credentials_file_path, blob_url):
+    """
+    Downloads the contents of a Google Cloud blob to a string.
```
Comment (lukechurch) [inline storage/google_cloud/google_cloud_utils.py]: s/Google Cloud/Google Cloud Storage/ 

(Google's platform has multiple different 'blobs' - this one is a GCS blob)

File: storage/google_cloud/google_cloud_utils.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                                           "URL (i.e. of the form gs://bucket-name/blob-name)"
+    bucket_name = parsed_blob_url.netloc
+    blob_name = parsed_blob_url.path.lstrip("/")
+
+    print(f"Downloading blob '{blob_name}' in bucket '{bucket_name}' to string...")
+    storage_client = storage.Client.from_service_account_json(bucket_credentials_file_path)
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+    blob_contents = blob.download_as_string().decode("utf-8")
+    print("Downloaded blob.")
```
Comment (lukechurch) [inline storage/google_cloud/google_cloud_utils.py]: Maybe report the length of blob_contents

File: storage/google_cloud/google_cloud_utils.py (hunk 1 above)
Comment (as2388) [reply storage/google_cloud/google_cloud_utils.py]: changed

File: storage/google_cloud/google_cloud_utils.py (hunk 2 above)
Comment (as2388) [reply storage/google_cloud/google_cloud_utils.py]: added a count to the print statement

**Comment by as2388** [comment]
Yes, just tested with "folder"s

**Pull request merged**